app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, Health:"
+                f" {self.health}, Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(victim: str) -> None:
+        if isinstance(victim, Herbivore):
```

Comment:
The `bite` method takes `victim` as a `str`, but it is used as if it were an instance of `Herbivore`. You should either change the parameter type to `Herbivore` or ensure that the method receives an actual `Herbivore` instance.